### DISCUSSION TITLE
When courses are shared from the app, the link that gets generated is redirected to the website and not to the app

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#476] [General Discussion]
Hello All,

**Scenario:**
I've shared a course from the Sunbird mobile app. Now, when I'm trying to open the generated link, it is getting opened in the chrome, not in the app.

Also, I'm not getting an option to whether open the link in the app or in the chrome.

I'm using the following deeplink:
**_deeplink_base_url = sunbirdsaas.com_**

In mobile, I'm having only one white labelled Sunbird App still not getting the option to whether open the link in the app or in the chrome.

**Can anyone help?**

cc: @swayangjit

**Comment by Abhign** [comment]
@swayangjit @vinukumar-vs Can you check this once.

CC: @Krishnaj20

**Reply by Ajoymaity** [reply]
@nidhikachhawa24 can you please test in other devices and share your device details.
It is getting opened in the app(samsung m30s, 11) and it's working fine for me.
Course url is : https://staging.sunbirded.org/explore-course/course/do_2136796955344240641444?referrer=utm_source%3Dmobile%26utm_campaign%3Dshare_content
CC: @swayangjit @vinukumar-vs

**Reply by swayangjit** [reply]
@nidhikachhawa24  Please let us know the OS version of the device that you are testing on .

**Reply by nidhikachhawa24** [reply]
@swayangjit 
I tried on 2 devices but facing same issue

OS details:

Xiaomi 11 lite ne , 13
Motorola moto G5, 8

**Reply by swayangjit** [reply]
@nidhikachhawa24  In Android 12+ devices the deeplink will not work in older releases, we have given the fix in release-5.2.0. You can find the fix [here](https://github.com/project-sunbird/cordova-plugin-sunbirdsplash/pull/39/files) and documentation is available [here](https://project-sunbird.atlassian.net/wiki/spaces/SBDES/pages/3276865567/Deep+link+support+in+Android+12+devices).

**Reply by nidhikachhawa24** [reply]
@swayangjit 
I've followed the steps from the doc that you've provided. But when I tried this in Android 13, it's still not working.

**Reply by swayangjit** [reply]
@nidhikachhawa24 are you able to generate the apk or you are generating via firebase ?

**Reply by nidhikachhawa24** [reply]
@swayangjit 
Yes I was able to generate the apk

I've changed the SHA key in assetlinks but still in android 13 the link is directly opening in browser

**Reply by swayangjit** [reply]
@nidhikachhawa24  Have you tried the fixes as discussed ?

**Reply by nidhikachhawa24** [reply]
@swayangjit 
I tried that but still it's not working

**Reply by swayangjit** [reply]
@nidhikachhawa24  Any updates on this? If this is not fixed we can schedule a call and see how to fix the issue

**Reply by nidhikachhawa24** [reply]
@swayangjit 
No, it's not fixed. We're actually migrating from 4.6 to 5.x soon

So will see if it's working there or not
